plugins/tiddlywiki/browser-storage/startup.js

```diff
@@ -19,19 +19,28 @@ exports.after = ["startup"];
 exports.synchronous = true;
 
 var ENABLED_TITLE = "$:/config/BrowserStorage/Enabled",
-	SAVE_FILTER_TITLE = "$:/config/BrowserStorage/SaveFilter",
-	QUOTA_EXCEEDED_ALERT_TITLE = "$:/config/BrowserStorage/QuotaExceededAlert",
-	DEFAULT_QUOTA_EXCEEDED_ALERT_PREFIX = "Quota exceeded attempting to store `",
-	DEFAULT_QUOTA_EXCEEDED_ALERT_SUFFIX = "` in browser local storage";
+	SAVE_FILTER_TITLE = "$:/config/BrowserStorage/SaveFilter";
+
+var BrowserStorageUtil = require("$:/plugins/tiddlywiki/browser-storage/util.js").BrowserStorageUtil;
 
 exports.startup = function() {
 	var self = this;
+	
+	// Add ENABLED tiddler with default value "yes"
+	$tw.wiki.addTiddler({title: ENABLED_TITLE, text: "yes"});
```

Comment:
- Did you test, if the configuration can be disabled?
- Is it possible to remove the configuration tiddler?

At the moment the browser storage plugin is disabled by default. @Jermolene ... Do we want to change that?

I think the plugin has to work with sensible defaults if _no_ config tiddler exists at all. That's the standard behaviour for all config tiddlers atm.

Reply:
Good catch. I added check for existence of the config tiddler. The browser storage plugin is actually enabled by default, but the save filter is very limited in scope. The current default value for the "Use browser local storage" checkbox is "yes", so the config tiddler is created to match. If you delete the config tiddler, the default for the checkbox once again applies.